src/ray/raylet/worker_pool.cc

```diff
@@ -378,7 +378,12 @@ WorkerPool::BuildProcessCommandArgs(const Language &language,
       if (!code_search_path.empty()) {
         path_env.append(":").append(code_search_path);
       }
-      env.emplace(kLibraryPathEnvName, path_env);
+      auto path_env_iter = env.find(kLibraryPathEnvName);
+      if (path_env_iter == env.end()) {
+        env.emplace(kLibraryPathEnvName, path_env);
+      } else {
+        env[kLibraryPathEnvName] = path_env_iter->second.append(":").append(path_env);
```

Comment:
Can we also append the library path which already is set to the system environment? E.g. `path_env_iter->second.append(":").append(path_env).append(os.environ["LD_LIBRARY_PATH"])`?

Reply:
Here, the "LD_LIBRARY_PATH" variable of the current environment variable has been set in it.
![image](https://user-images.githubusercontent.com/11072802/220239421-f7f85636-f21d-40fb-8d57-d4e6230d2848.png)